README.md

```diff
@@ -1,2 +1,22 @@
-# sublinks-docker
-docker-compose files to run Sublinks for both demo and local dev purposes
+# Sublinks Docker
+
+This repo will hold docker-compose.yml files to run Sublinks for both demo and local dev purposes
+
+CURRENT STATUS: Please note, this repo currently holds configuration files intended for development purposes only. As we get closer to our initial MVP release we will provide sample files for use in a live site.
+
+## Demo Site
+
+Run `docker-compose -f docker-compose.demo.yml up [-d]`
+
+## Frontend Dev
+
+Run `docker-compose -f docker-compose.frontend.yml up [-d]`
```

Comment:
`[-d]` aborts the command for me with `no matches found: [-d]`.
Command succeeds if I use `-d` without the brackets.

Reply:
The brackets indicate optional command arguments. I'll add a comment below each command to clarify

Reply:
TIL

Reply:
I updated the README. Please let me know if that helps?